bundler.d/test.rb

```diff
@@ -1,7 +1,7 @@
 group :test do
   gem 'mocha', '~> 1.11'
   gem 'single_test', '~> 0.6'
-  gem 'minitest', '~> 5.1'
+  gem 'minitest', '< 5.19'
```

Comment:
```suggestion
  gem 'minitest', '~> 5.1', '< 5.19'
```

I guess we still want to enforce the 5.1 lower bound

Reply:
(this was acked by Ewoud on IRC, so I am going to merge this once CI passes)